app/views/games/index.html.erb

```diff
@@ -1,182 +1,51 @@
 <div class="container-fluid">
-  <section id="register">
-    <div class= "row">
+  <section id="games-list">
+    <div class="row">
       <div class="col-sm-5 col-sm-offset-1">
```

Comment:
Offsetting one (`col-sm-offset-1`) on the first column ? I think maybe this should be on the second col in order to provide spacing between the two?